qiskit_nature/second_q/transformers/active_space_transformer.py

```diff
@@ -359,7 +359,9 @@ def _validate_num_orbitals(self, nelec_inactive: int, particle_number: ParticleN
 
     # TODO: can we efficiently extract this into the base class? At least the logic dealing with
     # recursion is general and we should avoid having to duplicate it.
-    def _transform_property(self, prop: Property) -> Optional[Property]:
+    def _transform_property(
+        self, prop: SecondQuantizedProperty
+    ) -> Optional[SecondQuantizedProperty]:
```

Comment:
Is this change truely necessary? Although this will be refactored further in the near future, this is technically incorrect because e.g. `DriverMetadata` is not a `SecondQuantizedProperty`

Reply:
Yes, I agree. I cannot remember what motivated the change, so I am reverting it.